client/containers/DashboardSubmissions/verdictDialog.scss

```diff
@@ -0,0 +1,24 @@
+.verdict-dialog {
+	.email-text-header {
+		text-transform: uppercase;
+		font-size: 12px;
+		color: #888;
+		margin-bottom: 5px;
+	}
+	.minimal-editor-component {
+			height: 150px;
+			overflow: auto;
```

Comment:
Do you happen to remember what problems the `overflow: auto`, `min-height: 100%`, and `white-space: break-spaces` solve? I have been trying for years to get this component to Do The Right Thing by default, and if you're having CSS frustrations around it that may be a sign that I should tweak something internally.

Reply:
`overflow: auto` keeps the formatting bar in view once the email copy runs long

`white-space: break-space` covers for when the placeholder text is longer than the input is wide. This is nearly impossible, and can be removed.

Can't recall why I included `min-height: 100%`, so I'll remove that too.